packages/react/src/SearchControl/SearchControl.tsx

```diff
@@ -92,6 +94,7 @@ interface SearchControlState {
   fieldEditorVisible: boolean;
   filterEditorVisible: boolean;
   filterDialogVisible: boolean;
+  exportEditorVisible: boolean;
```

Comment:
Can we rename this to `exportDialog` / `exportDialogVisible` ?

We use "Editor" for complex dialogs when the user is editing something, and "Dialog" for simple dialogs.